### DISCUSSION TITLE
Can stable and edge version installed at same time?

### DISCUSSION THREAD

**Discussion by hongbo-miao** [linkerd/linkerd2#6459] [Q&A] [answered]
I am wondering can stable and edge be installed at same time on local computer for testing purpose?
I didn't find the info at https://linkerd.io/edge/

If it is, how can we switch between versions? Something like`linkerd [command]` and `linkerd edge [command]`?

Or do we have to uninstall one and install another one?

**Comment by alpeb** [comment, marked as answer]
Definitely not, as linkerd installs cluster-level resources so you might get into inconsistencies between both versions. Uninstall+install is a quick operation so I'd recommend that for testing.
